### PR TITLE
Covid 19 banner

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/styles/_covid19.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.nhsuk-global-alert {
+  background-color: #ffeb3b;
+  padding-bottom: 24px;
```
Comment (alibombaye) [inline app/styles/_covid19.scss]: Should we be using the NHS-padding-bottom properties

File: app/styles/_covid19.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    font-size: 24px;
+    line-height: 1.33333;
+    margin: 0;
+  }
+
+  p {
+    font-size: 19px;
+    margin-top: 8px;
+    margin-bottom: 0;
+  }
```
Comment (alibombaye) [inline app/styles/_covid19.scss]: Same here should the NHS styling properties be used here instead if we can

File: app/includes/header.test.js [hunk 3]
```diff
@@ -112,5 +112,25 @@ describe('header', () => {
         });
       }));
     });
+
+    it('should render the covid19 global warning', createTestHarness(setup, (harness) => {
```
Comment (kat1906) [inline app/includes/header.test.js]: should have another test for should not show too (or add this to another test to make sure the banner is not shown)

File: app/styles/_covid19.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.nhsuk-global-alert {
```
Comment (kat1906) [inline app/styles/_covid19.scss]: we DO NOT start our own styles with `nhsuk-`. They must start with `bc-`

**Comment by kat1906** [comment]
Also this is 2 tickets in 1. Please do tickets one at a time no matter how small they are.

**Pull request merged**